frontend/styles/main.css

```diff
@@ -0,0 +1,47 @@
+body {
+    background-image:url(../components/Gameplay-background-image.jpg);
+
+}
+
+.BackgroundImage{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100;
+}
+
+.gameContainer{
+    display: flex;
+
+    justify-content: center;
+    vertical-align: top;
+
+    flex-direction: row;
+    grid-area: auto;
+}
+
+#gameCanvas{ 
```

Comment:
Don't worry too much for now, but generally we try and just use classes for css styling - when you're working on big projects, id's take precedence, and it's useful to have them to fall back on if some other styling is overruling the styling you've got!